qcengine/programs/turbomole/define.py

```diff
@@ -0,0 +1,209 @@
+from subprocess import Popen, PIPE, TimeoutExpired
+from typing import Any, Dict, Optional
+
+from .methods import METHODS, KEYWORDS
+
+
+def execute_define(stdin: str, cwd: Optional["Path"] = None) -> str:
+    # TODO: replace this with a call to the default execute provided by QCEngine
+    # if possible. May be difficult though, as we have to pipe in stdin and
+    # be careful with the encoding.
+
+    # We cant use univeral_newlines=True or text=True in Popen as some of the
+    # data that define returns isn't proper UTF-8, so the decoding will crash.
+    # We will decode it later on manually.
+    proc = Popen("define",
```

Comment:
`with Popen(...) as proc:`, best to open these as context so that we do not need to drop them.

Reply:
fixed in b18a244